packages/paste-core/components/progress-steps/stories/vertical.stories.tsx

```diff
@@ -102,3 +110,66 @@ export const Internationalized: React.FC = () => {
     </ProgressSteps>
   );
 };
+
+export const WithContent: React.FC = () => {
+  return (
+    <ProgressSteps orientation="vertical">
+      <ProgressStepComplete as="div">
+        Data warehouse connected
+        <ProgressStepContent>
+          <Box paddingX="space10">
```

Comment:
chore: You can remove the space10 paddingX from everything. That was just an unintentional Figma thing we're working out.